content/blog/2019/07/2019-07-30-jenkins-cli.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Managing Jenkins with jcli"
+tags:
+- general
+- jcli
+author: linuxsuren
+---
+
+As a developer, I usually use Jenkins like this:
+
+* Find a job which is related with your current work
+* Trigger that job
+* Check the output of log
+
+Sometimes, I might need to check the update center. Maybe a new plugin is needed, or update an exists plugin.
+Or, I want to upload a plugin from my computer. For all these cases, I just don't need a UI or even a browse.
+I like to use a CLI to complete most of my tasks. For example, I use `kubectl` to manage my Kubernetes cluster,
+to create or modify the kubernetes resources. So, I start to think that why not using a CLI to manage my Jenkins.
+
+== Why create a new one?
+
+First, I almost forget the Jenkins CLI is existed which is written by Java. Let me do dome introduce how to use the exist one.
```

Comment:
"First, I almost forgot about the existing Jenkins CLI, written in Java. Let me introduce how to use that one."